src/app/Models/Company/Employee.php

```diff
@@ -144,7 +144,7 @@ public function directReports()
      */
     public function employeeLogs()
     {
-        return $this->hasMany(EmployeeLog::class)->orderBy('audited_at', 'desc');
+        return $this->hasMany(EmployeeLog::class, 'author_id')->orderBy('audited_at', 'desc');
```

Comment:
It looks like you're currently trying to only show logs that the employee "authored" themselves instead of every log for that employee. This is a pretty big change from the current functionality, is this something the client requested?

Reply:
This is not something that the client specifically requested, but when I was reviewing the items reported by the client, I noticed that when looking at logs for a user, it was showing logs created by other users, which I assumed to be incorrect. This change has been removed.